pkgs/development/compilers/edk2/default.nix

```diff
@@ -11,6 +11,22 @@
 }:
 
 let
+# TODO: this needs to live somewhere else, provided by roberth
```

Comment:
@roberth I never did work out where to put this and otherwise this PR is ready to go, any ideas?

Reply:
Maybe in `lib/customisation.nix`?
I'm sorry that I didn't find this earlier.

Reply:
No worries, delaying getting this in isn't a big deal. Moved it there and marked ready for review :)